site/layouts/partials/footer.html

```diff
@@ -12,10 +12,10 @@ <h3 class="accordion-header footer-heading" id="headingLinks">
           </h3>
           <div id="collapseLinks" class="container-xxl accordion-collapse collapse" data-bs-parent="#sitemapAccordion">
             <ul class="navbar-nav">
-              <li><a class="nav-link" href="/" aria-describedby="headingLinks">Home</a></li>
-              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/" aria-describedby="headingLinks">Docs</a></li>
-              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/examples/" aria-describedby="headingLinks">Examples</a></li>
-              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/guidelines/" aria-describedby="headingLinks">Design guidelines</a></li>
+              <li><a class="nav-link{{ if .IsHome }} active" aria-current="page{{ end }}" href="/" aria-describedby="headingLinks">Home</a></li>
+              <li><a class="nav-link{{ if eq .Page.Layout "docs" }} active" aria-current="true{{ end }}" href="/docs/{{ .Site.Params.docs_version }}/" aria-describedby="headingLinks">Docs</a></li>
```

Comment:
shouldn't it be `aria-current="page"`like the others, since it is a set of pages?

Reply:
I used the same `aria-current` value than the values defined in `docs-navbar.html` for the navbar.
But yes I will double check with Vincent.